ModManager_Classes/GithubIntegration/StaticRepositoryInfoProvider.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Imya.GithubIntegration
+{
+    public class StaticRepositoryInfoProvider : IRepositoryInfoProvider
+    {
+        private Random random = new Random();
+
+        private GithubRepoInfo[] Repositories = {
+            new GithubRepoInfo { Name="Spice-it-Up", Owner = "anno-mods", AssetName = "Spice-it-Up.zip"},
+            new GithubRepoInfo { Name="New-World-Tourism", Owner="anno-mods", AssetName = "Gameplay.New.World.Tourism.zip" },
+            new GithubRepoInfo { Name="anno-1800-jakobs-mods", Owner="jakobharder", AssetName = "Gameplay.Modular.Factories.zip" },
+            new GithubRepoInfo { Name="anno-1800-jakobs-mods", Owner="jakobharder", AssetName = "Jakobs-Collection-v4.10.1.zip"},
+            new GithubRepoInfo { Name="WholesomeHaciendaHaul", Owner="Taludas", AssetName = "TheWholesomeHaciendaHaul_v1.3.2.zip"},
+            new GithubRepoInfo { Name="SmallModsCollection", Owner="Taludas", AssetName = "SmallModsCollection_v1.0.zip"},
+            new GithubRepoInfo { Name="Police_Station_MU", Owner="muggenstuermer", AssetName = "Source Code"},
+            
```

Comment:
Looks like we should define a repo top-level json file that says what the actual download file is?
Or maybe go for a fixed pattern like name or name+tag.

Reply:
okay, so since we don't know how we will do it later on, I simply gave the GithubRepoInfo a strategy to get the asset name, the rest is up to the future.